tasks.py

```diff
@@ -84,27 +87,34 @@ def task_wrapper(function=None):
 
 
 def docker_compose(context, command, **kwargs):
-    """Helper function for running a specific docker-compose command with all appropriate parameters and environment.
+    """Helper function for running a specific docker compose command with all appropriate parameters and environment.
 
     Args:
         context (obj): Used to run specific commands
-        command (str): Command string to append to the "docker-compose ..." command, such as "build", "up", etc.
+        command (str): Command string to append to the "docker compose ..." command, such as "build", "up", etc.
         **kwargs: Passed through to the context.run() call.
     """
-    compose_command = f'docker-compose --project-name {context.nautobot.project_name} --project-directory "{context.nautobot.compose_dir}"'
+    compose_command_tokens = [
+        "docker compose",
```

Comment:
The only issue I see with this is for remote Docker setups. Example: If I'm running Docker in the cloud or server elsewhere, the standalone Docker compose script is still `docker-compose`. `docker compose` AFAIK only works if you have Docker Desktop installed.

@whitej6 Does this ring a bell to you?

Reply:
https://docs.docker.com/compose/install/compose-plugin/#install-the-plugin-manually

> **Compose standalone:** If you need to use Compose without installing the Docker CLI, the instructions for the standalone scenario are similar. Note the target folder for the binary’s installation is different as well as the compose syntax used with the plugin (space compose) or the standalone version (dash compose).

Reply:
I'll change it back to `docker-compose` then. I just don't like that it's a much older version of the software, at least under Docker Desktop:

```
> docker-compose --version
docker-compose version 1.29.2, build 5becea4c
> docker compose version
Docker Compose version v2.2.3
```

Reply:
Yeah my thought was more around the setups where one may be using V2 but it's only available as `docker-compose` because they chose to not install the entire Docker CLI, just the Docker Compose standalone.

Maybe a `compose_command` context variable that defaults to `docker compose` but can be overridden if necessary.

Reply:
Actually just ran into this last night in a fresh docker deployment for a demo environment I spun up.